app/controllers/login.js

```diff
@@ -0,0 +1,32 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { match, not } from '@ember/object/computed';
+
+export default class LoginController extends Controller {
+  @service session;
+  @service router;
+  @tracked email;
+  @tracked password;
+  @tracked error;
+
+  @action
+  onChange(event) {
+    this.set(event.target.name, event.target.value)
+  }
+
+  @action
+  async authenticate(event) {
+    event.preventDefault();
+    const email = this.get('email');
+    const password  = this.get('password');
+    try {
+      await this.session.authenticate('authenticator:oauth2', email, password);
+      this.router.transitionTo('points');
```

Comment:
why do you need this when you've already defined `routeAfterAuthentication` as `points`?

Reply:
I am getting error from routes, and it not redirecte to points route so i did this, i will remove it and try to fix it.
Thank You